src/Interpreters/Context.cpp

```diff
@@ -511,6 +511,9 @@ struct ContextSharedPart : boost::noncopyable
     std::atomic_size_t max_view_num_to_warn = 10000lu;
     std::atomic_size_t max_dictionary_num_to_warn = 1000lu;
     std::atomic_size_t max_part_num_to_warn = 100000lu;
+    // these variables are used in inserting warning message into system.warning table based on asynchronous metrics
+    size_t max_pending_mutations_to_warn = 100lu;
+    size_t max_stuck_mutations_to_warn = 100lu;
```

Comment:
Default values here don't match default values in src/Core/ServerSettings.cpp. Btw, do we really need these variables and getters/setters for them, can we just get it from server settings?

Reply:
Fixed the default values. We can get rid of the getters and setters. But right now other similar settings for warning use them, so keeping it. I'll follow up to remove them.